pytests/src/pyfunctions.rs

```diff
@@ -67,13 +67,21 @@ fn args_kwargs<'py>(
     (args, kwargs)
 }
 
-#[pymodule(gil_used = false)]
-pub fn pyfunctions(m: &Bound<'_, PyModule>) -> PyResult<()> {
-    m.add_function(wrap_pyfunction!(none, m)?)?;
-    m.add_function(wrap_pyfunction!(simple, m)?)?;
-    m.add_function(wrap_pyfunction!(simple_args, m)?)?;
-    m.add_function(wrap_pyfunction!(simple_kwargs, m)?)?;
-    m.add_function(wrap_pyfunction!(simple_args_kwargs, m)?)?;
-    m.add_function(wrap_pyfunction!(args_kwargs, m)?)?;
-    Ok(())
+#[pyfunction(signature = (a, /, b))]
```

Comment:
It might be nice to add tests / benchmarks to `test_pyfunctions.py` to match the rest of the functions in this file.

Reply:
Done